dev/cli

```diff
@@ -47,7 +47,7 @@ GLOBAL OPTIONS
     --help                                    - Show this message
 COMMANDS
     -gce     Google compute engine instance name
-    -gcf     Google cloud function URL (Predefine ci/authn-gcp/function/main.py code as GCF)
```

Comment:
You have some mistakes in this merge, you took the old version of the file

Reply:
why is this `-gce` and not `--gce`? same for `gcf`

Reply:
because is a sub parameter for `--authn-gcp` so i wanted is to be different , but i dont have s storing opinion it could be also `--gce` and `--gcf`